src/generate-changelog.ts

```diff
@@ -10,7 +10,8 @@ const TEAMS_SPECIFIC_REGEX = /^TEAMS-SPECIFIC-CHANGES:(.*)/gm;
 // Template text for the changelog that should be ignored.
 const CHANGELOG_TEMPLATE_TEXT = [
   '{{Insert a single changelog entry here}}',
-  '{{Insert ANOTHER changelog entry here}}'
+  '{{Insert ANOTHER changelog entry here}}',
```

Comment:
we probably should just drop anything that is matches `{{.*}}` at this point

Reply:
good idea